lib/active_model/entity/meta/descriptions.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module ActiveModel
+  module Entity
+    module Meta
+      # Provides helper routines allowing ActiveModel::Entity to specify descriptions on the attributes and the entity.
+      module Descriptions
+        extend ActiveSupport::Concern
+
+        included do
+          class_attribute :meta_descriptions, default: {}
```

Comment:
maybe just add `nil: []` into initial value and remove all other checks?

Reply:
done